src/datagrid/DataGrid.tsx

```diff
@@ -219,7 +219,7 @@ type DataGridPaginationProps = {
     style?: any;
     currentPage?: number;
     pageSize?: number;
-    pageSizes?: number[];
+    pageSizes?: any[];
```

Comment:
```suggestion
    pageSizes?: string[];
```

Reply:
type of pageSizes is set as any because other stories pass number array as prop. Setting this to string results in errors